examples/launcher/src/components/Engine.jsx

```diff
@@ -220,6 +208,17 @@ class Engine extends React.Component {
       });
     }
 
+    onConsoleClick() {
+      let studioConsole = document.getElementById('console');
+      if(studioConsole.clientHeight === 0){
+        studioConsole.style.height = "100px";
```

Comment:
This is better done with a `.open` class + css.

Once we start getting into the game of sizing things in the Javascript it's prone to quickly devolve into nightmare math.

Reply:
b7eeba5 61b6e79 99af7bf